.github/workflows/ci.yml

```
@@ -59,6 +60,14 @@ jobs:
         with:
           github-token: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
           no-restore: true # don't download them on a cache hit
+      # control checking out head instead of ref by a GH label
+      # if checkout-head label is added to a PR, checkout HEAD otherwise checkout ref
+      - if: ${{ !contains(github.event.pull_request.labels.*.name, 'checkout-head') }}
+        run: echo "CHECKOUT_REF=${{ github.base_ref }}" >> "$GITHUB_ENV"
```

```suggestion
        run: echo "CHECKOUT_REF=${{ github.ref }}" >> "$GITHUB_ENV"
```
backport the fix too!

---

I'm confused, I see this was resolved, but I still see `base_ref` here 🤔 am I holding this thing wrong?

---

Nope, I was holding it wrong, haha.  Too many tabs open, pushed to the other backport by accident.